From: Sven Roederer <devel-sven@geroedel.de>
Date: Tue, 13 Apr 2021 01:09:50 +0200
Subject: ath79: disable default build for all compex devices

* there is a failure in the mkmylofw tool filling to the disk.
* see http://lists.openwrt.org/pipermail/openwrt-devel/2021-April/034680.html

diff --git a/target/linux/ath79/image/generic.mk b/target/linux/ath79/image/generic.mk
index 303907e5bb788ffc6d8d2cb31874909609a7a4fd..6bdbbfe64214a70e797a7079eccae8aa275cc393 100644
--- a/target/linux/ath79/image/generic.mk
+++ b/target/linux/ath79/image/generic.mk
@@ -623,6 +623,7 @@ define Device/compex_wpj344-16m
   SUPPORTED_DEVICES += wpj344
   IMAGES += cpximg-6a08.bin
   IMAGE/cpximg-6a08.bin := append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | mkmylofw_16m 0x690 3
+  DEFAULT := n
 endef
 TARGET_DEVICES += compex_wpj344-16m
 
@@ -639,6 +640,7 @@ define Device/compex_wpj531-16m
   IMAGE/cpximg-7a04.bin := append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | mkmylofw_16m 0x693 3
   IMAGE/cpximg-7a06.bin := append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | mkmylofw_16m 0x693 3
   IMAGE/cpximg-7a07.bin := append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | mkmylofw_16m 0x693 3
+  DEFAULT := n
 endef
 TARGET_DEVICES += compex_wpj531-16m
 
@@ -651,6 +653,7 @@ define Device/compex_wpj563
   SUPPORTED_DEVICES += wpj563
   IMAGES += cpximg-7a02.bin
   IMAGE/cpximg-7a02.bin := append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | mkmylofw_16m 0x694 2
+  DEFAULT := n
 endef
 TARGET_DEVICES += compex_wpj563
 
